apps/src/dance/lab2/views/DanceView.tsx

```diff
@@ -260,8 +274,15 @@ const DanceView: React.FunctionComponent<{
       resetProgram();
       // TODO: Handle puzzle complete.
       console.log(`onPuzzleComplete! pass?: ${result} message: ${message}`);
+      if (result) {
+        currentCondition.current = {name: 'pass'};
```

Comment:
Minor: could we just have a setCurrentCondition() on `DanceValidator` instead of saving off the last condition to a ref?